datastore/storagebackend/postgresql/putobservations.go

```diff
@@ -370,6 +373,14 @@ func (sbe *PostgreSQL) PutObservations(request *datastore.PutObsRequest) error {
 			return fmt.Errorf("getObsTime() failed: %v", err)
 		}
 
+		if obsTime.AsTime().Before(loTime) {
+			return fmt.Errorf("obs time too old: %v < %v", obsTime.AsTime(), loTime)
+		}
+
+		if obsTime.AsTime().Before(loTime) {
```

Comment:
This should be the check "after hiTime", correct?

Reply:
Of course! Wll fix.